plugin/src/main/java/com/gtnewhorizons/retrofuturagradle/modutils/UpdateDependenciesTask.java

```diff
@@ -25,7 +25,7 @@
 
 public class UpdateDependenciesTask extends DefaultTask {
 
-    private static final Pattern GTNH_DEPENDENCY = Pattern.compile("com\\.github\\.GTNewHorizons:(.+):(.+):dev");
+    private static final Pattern GTNH_DEPENDENCY = Pattern.compile("com\\.github\\.GTNewHorizons:(.+):(.+):.+");
```

Comment:
This does not allow empty classifiers, make the last `:.+` optional.

Reply:
regex became a bit more complicated but it should work